includes/abstracts/abstract-wc-stripe-connect-rest-controller.php

```diff
@@ -42,41 +44,41 @@ public function register_routes() {
 			register_rest_route(
 				$this->namespace,
 				'/' . $this->rest_base,
-				array(
```

Comment:
This old `array()` syntax is needed for supporting PHP 5.2. Is it possible not to replace this syntax until a decision is made to no longer support PHP 5.2?

Reply:
We can disable that rule... 
And we should also change the minimum compatibility check to be WP 4.7 with PHP 5.2 (instead of 5.6)

Reply:
Updated the phpcs config to:
- min wp version: 4.7
- php test version: 5.6- (minimum supported by trunk)